activiti-core/activiti-engine/src/main/resources/org/activiti/db/upgrade/activiti.mysql55.upgradestep.800.to.810.history.sql

```diff
@@ -0,0 +1 @@
+alter table ACT_HI_IDENTITYLINK add column DETAILS_ LONGBLOB;
```

Comment:
Missing migration from `mysql`? I only see `mysql55`

Reply:
Mysql updgrade exists here: https://github.com/Activiti/Activiti/pull/4444/files#diff-00b1b0e8c4ec776b678c0eea023a4e9faad83407da3057d95e46344d37323df3R1

Reply:
The one you mentioned is `mssql`, not `mysql` :D